src/egon/data/datasets/power_plants/wind_offshore.py

```diff
@@ -212,42 +231,41 @@ def insert():
         )
 
         # Drop NANs in column osm_id
-        busses.dropna(subset=["osm_id"], inplace=True)
+        buses.dropna(subset=["osm_id"], inplace=True)
 
         # Create columns for bus_id and geometry in the offshore df
-        offshore["bus_id"] = 0
+        offshore["bus_id"] = pd.NA
```

Comment:
I had to change this in the last PR from nan to 0. Otherwise, the datatype was `float` which created problems later on. Was that not the case for you?